cloudformation/kms.yaml

```diff
@@ -50,22 +41,13 @@ Resources:
           - Sid: 'Enable IAM Root Permissions'
             Effect: Allow
             Principal:
-              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+              AWS: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:root'
             Action: 'kms:*'
             Resource: '*'
-          - Fn::If:
-              - isDev
-              - Sid: 'Enable IAM User Permissions'
-                Effect: Allow
-                Principal:
-                  AWS: ${self:custom.config.devAwsUserAccountArn}

```

Comment:
@swoldemi are you removing this stuff since it can't be determined by cloudformation?

Reply:
I removed it on @carvantes's recommendation in [an earlier comment](https://github.com/awslabs/fhir-works-on-aws-deployment/pull/121#discussion_r504298286)

Reply:
Yep thanks for the clarification